tests/notifier/test_telegram_notifier.py

```diff
@@ -0,0 +1,37 @@
+# std
+import os
+import unittest
+
+# project
+from src.notifier import Event, EventType, EventPriority, EventService
+from src.notifier.telegram_notifier import TelegramNotifier
+
+
+class TestTelegramNotifier(unittest.TestCase):
```

Comment:
I would add some tests for the HIGH and NORMAL priority cases, if they get handled differently.